mm2src/coins/eth.rs

```diff
@@ -1258,7 +1106,10 @@ impl SwapOps for EthCoin {
         MmError::err(ValidateInstructionsErr::UnsupportedCoin(self.ticker().to_string()))
     }
 
-    fn is_supported_by_watchers(&self) -> bool { true }
+    fn is_supported_by_watchers(&self) -> bool {
+        false
+        //self.contract_supports_watchers
```

Comment:
Why it's hardcoded as false?

Reply:
The ETH watcher rewards aren't complete yet, and watchers shouldn't be used until it's ready